packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -33,4 +34,19 @@ export const initNPM = async (dir: string, task: ForgeListrTask<any>): Promise<v
     task.output = `${packageManager} install --dev --exact ${packageName}`;
     await installDepList(dir, [packageName], DepType.DEV, DepVersionRestriction.EXACT);
   }
+
+  // This logic allows developers working on forge itself to easily init
+  // a local template and have it use their local plugins / core / cli packages
```

Comment:
This is awesome! Any specific command devs can use to do this - is it `LINK_FORGE_DEPENDENCIES_ON_INIT=true` or running `link:prepare`? Just wondering if this is something we can document for future use.

Reply:
You run `link:prepare` and then when you call `electron-forge-init.js` manually you have this env var set so it links correctly